### PR TITLE
Small Chimes fix

### PR DESCRIPTION
**Pull request by Elements6007**
Fixes the cut off "s" at the end of "mins" in the Chimes settings. 
InkedPXL_20221030_035021069small

^"s" is cut off

PXL_20221030_033625363small

^"s" is now fully visible

### CODE REVIEW

**Review by JF002 — COMMENTED** [review]
Looks good to me, thanks! 

Would you like to have a look and check why the code formatting workflow failed during the build?

**Comment by Elements6007** [comment]
> Looks good to me, thanks!
> 
> Would you like to have a look and check why the code formatting workflow failed during the build?

I forgot to run the Clang format because it was just a small edit, it should be fixed now. Thanks!

File: src/displayapp/screens/settings/SettingChimes.h [hunk 1]
```diff
@@ -23,16 +23,14 @@ namespace Pinetime {
...
+                                                           {Controllers::Settings::ChimesOption::HalfHours, "Every 30 mins"}}};
 
         std::array<lv_obj_t*, options.size()> cbOption;
 
         Controllers::Settings& settingsController;
       };
     }
   }
-}
+}
```
Comment (NeroBurner) [inline src/displayapp/screens/settings/SettingChimes.h:36]: Does the clang format actually require no newline at the end of the file? I don't think so, could you please check and revert

```suggestion
}

```

**Review by NeroBurner — CHANGES_REQUESTED** [review]

**Review by NeroBurner — APPROVED** [review]
Looking good now, thanks for the fix!

InfiniSim_2022-11-06_205140

**Review by JF002 — APPROVED** [review]

**Comment by JF002** [comment]
Thanks !

**Pull request merged**